src/lib/ddpConnector.ts

```diff
@@ -217,7 +147,7 @@ export class DDPConnector extends EventEmitter {
 			this._connected = connected
 
 			if (connected) {
-				this._connectionId = this.ddpClient.session
+				this._connectionId = this.ddpClient && this.ddpClient.session
```

Comment:
These uses of `this.ddpClient` could also use the `this.ddpClient?.session` syntax

Reply:
Was introduced in typescript 3.7 and so this makes sense. Done in commit https://github.com/nrkno/tv-automation-server-core-integration/pull/34/commits/5d460512757ac9a4f444b8315ffb59a592d10da1

Reply:
Now an option due to Typescript >3.7, so yes :-) Done in commit https://github.com/nrkno/tv-automation-server-core-integration/pull/34/commits/5d460512757ac9a4f444b8315ffb59a592d10da1